pinlist.yaml

```diff
@@ -149,3 +149,9 @@
   meta:
     tags:
     - all
+- cid: QmPmzUrGNpAUNM3H6vpeWS5pyjktzBf6PyahScPGgsyy3C
+  name: ORCESTRA HEAD v3
+  prev: QmWJbdYZWoea5eNY7vNxHH6MzhXnbzorsATFeJhtFxsSFN
```

Comment:
To stay compatible with the pinning api spec, I'd probably move the prev into meta.
I'd probably also retroactively add the prev links for earlier versions, as that might help speeding up pinning on new, not yet existing nodes.

Reply:
Done.